Hardcore.lua

```diff
@@ -2065,6 +2019,103 @@ function Hardcore:SetGriefAlertCondition(grief_alert_option)
 	end
 end
 
+function Hardcore:DKConvert(dk_convert_option) 
```

Comment:
nit: since most code is in Hardcore.lua conflicts are becoming more common. We should start separating files a bit more so to avoid conflicts. Not required, just a nice to have change.

Reply:
Yeah. I'll move next changes to separate lua file so it's easier to merge.